### ISSUE TITLE
create video mixer module

### ISSUE THREAD

**Issue by stephenwhitmarsh** [eegsynth/eegsynth#313]
Since I have started with learning how to create GUI's and work with visual elements and PyQt especially, as well as the fact that the videomixer software is only used for basic mixing, I will make a videomixer module with options that can be created in conversation with Per. Basic functionalities will be mixing according to Redis values, as well as some basic effects such as hue, speed, etc.

**Comment by robertoostenveld** [comment]
I don't think it is very efficient to work on eegsynth software modules for which we already have external solutions (as avmixer). You are bound to run into hard but non-interesting issues like video codecs and file formats. Furthermore, avmixer supports [syphon](http://syphon.v002.info), allowing for interesting feedback options.
